test/benchgc/src/benchgc/pattern/mlp.py

```diff
@@ -0,0 +1,315 @@
+                        if bias:
+                            broadcast_bias = linalg.broadcast(
+                                bias,
+                                outs=[tensor.EmptyOp(layer_out_shape, dtype)],
+                                dimensions=[0],
+                            )
+                            data = linalg.add(
+                                data,
+                                broadcast_bias,
+                                outs=[tensor.EmptyOp(layer_out_shape, dtype)],
+                            )
+
+                        if self.act_type == "relu":
+                            element = FloatAttr.get(dtype, 0)
+                            tensor_type = ir.RankedTensorType.get(
+                                layer_out_shape, dtype
+                            )
+                            attr = ir.DenseElementsAttr.get_splat(tensor_type, element)
+                            cst = arith.ConstantOp(tensor_type, attr)
+                            data = linalg.max(
+                                data, cst, outs=[tensor.EmptyOp(layer_out_shape, dtype)]
+                            )
+                    func.ReturnOp([data])
+        return module
+
+    def prepare_np_args(self, disable_results_to_params: False) -> List[np.ndarray]:
+        bench_func = get_kernel_func_from_module(self.ir_module, self.main_entry)
+        np_args = []
+        for arg in bench_func.arguments:
+            np_args.append(make_mlir_ndarray(arg.type))
+
+        if not disable_results_to_params:
+            for res in bench_func.type.results:
+                np_args.append(make_mlir_ndarray(res))
+        return np_args
+
+    @staticmethod
+    def add_args(parser: argparse.ArgumentParser):
+        parser.add_argument("--batch_size", type=int, default=1)
+        parser.add_argument("--hidden_size_list", type=str, default="")
+        parser.add_argument("--has_bias", required=False, type=str)
+        parser.add_argument(
+            "--act_type", type=str, choices=["noop", "relu", "sigmoid"], default="noop"
+        )
+        parser.add_argument(
+            "--dtype",
+            type=str,
+            choices=[
+                "f32",
+                "bf16",
+            ],
+            default="f32",
+        )
+
+    def handle_args(self, args: argparse.Namespace):
+        self.batch_size = args.batch_size
+        assert self.batch_size > 0, "batch size should be greater than 0"
+
+        self.hidden_size_list = to_int_list(args.hidden_size_list)
+        layers = len(self.hidden_size_list) - 1
+        assert layers >= 1, "hidden_size_list should have at least 2 elements"
+
+        self.has_bias = (
+            [False] * layers if args.has_bias is None else to_bool_list(args.has_bias)
+        )
+
+        assert (
+            len(self.has_bias) == layers
+        ), "has_bias should have the same length as hidden_size_list"
+
+        self.act_type = args.act_type
+        self.dtype = args.dtype
```

Comment:
@xurui1995 
This part is not compatible with the argument design in benchgc now.
Need to make a big change